.github/workflows/holochain-build-and-test.yml

```diff
@@ -84,3 +76,52 @@ jobs:
           #   export basecmd="nix run -L --show-trace .#packages.${system}."
           #   ${basecmd}${{ join(matrix.cmd.pkgs, '; ${basecmd}') }}
           # fi
+
+  # ensures the cache is regularly updated for the supported versions on multiple platforms
+  cache-update:
+    # This only runs on develop because it points at develop anyway, no need for maintenance branches to do the same job
+    # if: github.ref == 'refs/heads/develop'
```

Comment:
Need to re-enable. Build on develop are frequent enough because develop will always build after a source update.

Something we're not doing though is updating the versions flakes after an overlay update. Do we need to add that for this to make sense to re-run so frequently?